index.html

```diff
@@ -12,18 +12,18 @@
     <main id="main">
         <div id="first">
         <h1 id="title">Dr. Norman Borlaug</h1>
-        <p>The man who saved a billion lives</p>
+        <p>The man who saved a billions lives</p>
     </div>
         <div id="img-div">
             <img id="image" src="https://www.w3schools.com/images/picture.jpg" alt="image" class="center">
             <div id="img-caption">
                 Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields -
-                part of his life-long war on hunger.
+                parts of his life-long war on hunger.
             </div>
         </div>
         <div id="tribute-info">
             <h3>
-                Here's a time line of Dr. Borlaug's life:
+                Here's a time line of Dr. Borlaug's life:y
```

Comment:
This is a great job. but you should try using class as your CSS selector and avoid id selector
example: <header class="header">

Reply:
- [ ]  Good job expressing your  appreciation to the student's work, I think it would be better if you suggested your opinion instead of giving it as a must. :recycle: 

I suggest ot say _"This is a great job. I suggest using class as your CSS selector and avoid id selector
example:'_  instead of  _"This is a great job. but you should try using class as your CSS selector and avoid id selector
example:"_